src/main/java/com/dev/cinema/controllers/MovieSessionController.java

```diff
@@ -0,0 +1,49 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.dto.moviesession.MovieSessionDtoMapper;
+import com.dev.cinema.model.dto.moviesession.MovieSessionRequestDto;
+import com.dev.cinema.model.dto.moviesession.MovieSessionResponseDto;
+import com.dev.cinema.service.MovieSessionService;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    @Autowired
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @GetMapping("/available")
+    @ResponseBody
+    public List<MovieSessionResponseDto> getAllAvailableMovieSession(
+            @RequestParam("movieId") Long movieId,
+            @RequestParam("date") LocalDate date) {
+        return movieSessionService.findAvailableSessions(movieId, date)
+                .stream()
+                .map(movieSessionDtoMapper::mapMovieSessionResponseDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping("/")
```

Comment:
@PostMapping("/") is the same with @PostMapping

Reply:
ok. agree